openmls/src/key_packages/mod.rs

```diff
@@ -157,7 +157,9 @@ pub use lifetime::Lifetime;
 ///     Extension extensions<V>;
 /// } KeyPackageTBS;
 /// ```
-#[derive(Debug, Clone, PartialEq, TlsSize, TlsSerialize, Serialize, Deserialize)]
+#[derive(
+    Debug, Clone, PartialEq, TlsSize, TlsSerialize, Serialize, Deserialize, TlsDeserialize,
```

Comment:
These structs don't implement `deserialize` on purpose. Why do you need these?
The `VerifiableKeyPackage` is handling the deserialization to enforce validation and signature checks.

Reply:
Good catch. I initially had these because I had planned to serialize everything before returning to js, but (a) it looks like i was doing that wrong and (b) we use opaque objects now instead of serializing so it doesn't really matter.